ambersim/logger/logger.py

```diff
@@ -0,0 +1,146 @@
+import jax
+import wandb
+from torch.utils.tensorboard import SummaryWriter
+
+
+class BaseLogger:
+    """Base logger interface that defines common methods for logging metrics and parameters.
+
+    Attributes:
+        log_dir (str): Directory where logs are stored. If None, default locations are used.
+    """
```

Comment:
```suggestion
    """Base logger interface that defines common methods for logging metrics and parameters."""
```
nit: this is redundant with `__init__`.